library/CMakeLists.txt

```diff
@@ -176,11 +176,17 @@ if(F3D_STRICT_BUILD AND MSVC)
   target_compile_definitions(libf3d PRIVATE _SILENCE_ALL_CXX17_DEPRECATION_WARNINGS)
 endif()
 
+if(APPLE)
+  set_target_properties(libf3d PROPERTIES INSTALL_RPATH "@loader_path")
+elseif(UNIX)
+  set_target_properties(libf3d PROPERTIES INSTALL_RPATH "$ORIGIN")
+endif()
```

Comment:
what makes it needed now ?

Reply:
It's not needed, but while investigating on the crash in the CI I noticed this issue. Makes more sense now in my opinion. If you want I can put it in another PR.

Reply:
So without it we can have issues ?

Reply:
Yes, if you want to create a relocatable installation with all dependencies (without the SB), you will install `libVTKxxx.so` for example in `/opt/f3d_relocatable/lib` and `libf3d.so` in the same folder and you want it to work fine if you rename the folder `/opt/f3d_newname`, then it will not work.  
I think we don't have the problem in the SB because the SB is patching the RPATH.

Reply:
got it.